### DISCUSSION TITLE
Why should I use the Map Capacitor plugin with my next Ionic/Angular native app

### DISCUSSION THREAD

**Discussion by Sampath-Lokuge** [ionic-team/capacitor#5591] [Q&A] [answered]
Hi,

I can see that Ionic has a [Google map Capacitor plugin](https://capacitorjs.com/docs/apis/google-maps). i.e. it uses native Google Maps SDK. But my question here is why should I use it instead of the [Angular Google Maps component](https://github.com/angular/components/blob/main/src/google-maps/README.md). 

I cannot see any price advantage there too. Actually, Javascript API has more free usage and etc too.

Price: https://mapsplatform.google.com/pricing/

So then why should I use the Map Capacitor plugin with my next Ionic/Angular native apps?

**Comment by mlynch** [comment, marked as answer]
If you're happy with the web-based Maps SDK, then continue to use that. For some applications the native maps SDK will work and feel better. Additionally, it can be easier to configure and set up the native maps SDK on mobile. Whatever works for you.

**Reply by Sampath-Lokuge** [reply]
Yes, I'm a master of that Angular component. Thanks a lot for the clarification!

**Reply by mlynch** [reply]
There's some additional nuance here. The Native SDK supports a free mode that they make very difficult to understand, while the web one doesn't. More info here: https://developers.google.com/maps/billing-and-pricing/pricing#mobile-dynamic

**Reply by Sampath-Lokuge** [reply]
Nice to know about it too. It is always good to have a lot of options here. i.e. native and web options. I think that is the power of using a web tech stack for developing native apps. Thanks again!
